helm/overlay/kustomization.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 # import the default deployment as the base
-bases:
```

Comment:
last time @mowies had concerns in changing this statement

Reply:
that's ok, we should change that for kustomize v5 anyways